### PR TITLE
Update ideas.html

### PR DESCRIPTION
**Pull request by mstechly**

### CODE REVIEW

File: ideas.html [hunk 1]
```diff
@@ -92,6 +92,18 @@
...
                         </ul>
                     </p>
+            
+                    <p>
+                        <a href="www.mustythoughts.com">Michał Stęchły </a> - an Advisory Board member from
+                        <a href="https://www.zapatacomputing.com/">Zapata Computing</a> -
+                        shares his ideas:
+                        <ul style="list-style-type: none;">
+                        <li class="leading-block">Visualization tools for high-dimension optimization landscapes could help get better insight into algorithms like QAOA or VQE.</li><br>
+                        <li class="leading-block">The number of new publications showing up on arxiv is overwhelming. Some kind of an automated knowledge aggregator would be extremely useful for the researchers.</li><br>
```
Comment (willzeng) [inline ideas.html:102]:
```suggestion
                        <li class="leading-block">The number of new publications showing up on arxiv can be overwhelming. Some kind of an automated/community knowledge aggregator could greatly assist in research.</li><br>
```

File: ideas.html [hunk 2]
```diff
@@ -92,6 +92,18 @@
...
+                        <li class="leading-block">At QOSF we have done <a href="https://qosf.org/evaluation/">evaluation of open source projects</a> in quantum in 2018. Help with automating the process would be appreciated.</li><br>
```
Comment (willzeng) [inline ideas.html:103]:
```suggestion
                        <li class="leading-block">At QOSF we have done <a href="https://qosf.org/evaluation/">evaluation of open source projects</a> in quantum in 2018. Help with updating and automating the process would be appreciated.</li><br>
```

**Comment by willzeng** [comment]
@mstechly Thanks so much for these great suggestions! I made a few small tweaks to the wording. Let me know if those look good and then we can add them in and merge.

**Comment by mstechly** [comment]
Thanks for these, good to merge!

**Review by willzeng — APPROVED** [review]

**Pull request merged**